cli/install.py

```diff
@@ -44,8 +44,7 @@ def get_args(args=None):
     :return args: return loaded arguments from CLI
     """
 
-    spec_manager = conf.SpecManager(CONF)
-    args = spec_manager.parse_args(ENTRY_POINT, args=args)
+    args = conf.SpecManager.parse_args(ENTRY_POINT, CONF, args=args)
```

Comment:
This method also encloses the ENTRY_POINT and CONF module variables. I would prefer to leave method as is. Otherwise we need to add additional imports in all the tests were this method is used.